src/v0/destinations/facebook_pixel/transform.js

```diff
@@ -246,7 +246,10 @@ const responseBuilderSimple = (message, category, destination, categoryToContent
     if (!isActionSourceValid) {
       throw new InstrumentationError('Invalid Action Source type');
     }
+  } else {
+    commonData.action_source = 'other';
```

Comment:
Add documentation reference. Also, some explanation about this choice.

Reply:
We will do the follow-up task on that to correctly configure the action source (for JS sdk, mobile and other)

Reply:
Could you add some details about the fix in the PR?

Reply:
added